MultiLoggerSingleton/Logger.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.IO;
+
+namespace MultiLoggerSingleton
+{
+    public class Logger: ILogger
+    {
+        public static int m_count = 0;
+        private readonly string m_logFile;
+        private static Logger instance = null;
+        private static readonly object lockobject  = new object();
+
+        // private constructor
+        private Logger() 
+        { 
+            m_count++;
+            Console.WriteLine($"Constructor is called for {m_count} time.");
+            m_logFile = "D:\\GitHub\\DesignPatterns\\MultiLoggerSingleton\\logFile.txt";
```

Comment:
Hard coded path

Reply:
removed hard coded path from Logger class and added it in Main( ) and passing as argument to LogToFile ( ).